index.js

```diff
@@ -8274,28 +8274,48 @@ async function main() {
 
   console.log({ eventName, sha, headSha, branch, owner, repo });
   const workflow_id = core.getInput('workflow_id', { required: true });
+  const workflow_ids = workflow_id.replace(/\s/, '').split(',');
```

Comment:
```suggestion
  const workflow_ids = workflow_id.replace(/\s/g, '').split(',');
```
There might be more than one space that needs to be replaced.